pkg/virt-operator/resource/apply/reconcile.go

```diff
@@ -617,7 +617,7 @@ func (r *Reconciler) createOrRollBackSystem(apiDeploymentsRolledOver bool) (bool
 	// create/update API Deployments
 	for _, deployment := range r.targetStrategy.ApiDeployments() {
 		deployment := deployment.DeepCopy()
```

Comment:
We are talking about an oversight from previous work, right? I don't think I introduced a new copy.
I'm happy to do some cleanup though! What would you think about removing the copy from `syncDeployment` instead, since it would also address your other comment and simplify this PR?

Reply:
oh, yeah i see what you're saying. you didn't make any new change here, the name of the deployment in syncDeployment simply changed to origDeployment.

Since this isn't new, lets just leave it the way it is unless you're wanting to address it.